docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -5,39 +5,53 @@
 :sectnums:
 :icons: font
 
-Using Java's Streams API,
+Using Java's Streams API or Python's generator expressions,
```

Comment:
This is another example where maybe talking about a particular language doesn't make sense anymore.

This can be "Let's implement ... using a functional approach:".
Then inside the tab for a particular language, we can add more details specific to that language, if we need to.